src/Form/index.tsx

```diff
@@ -683,14 +684,15 @@ function Form({
         // @ts-expect-error TS(2345): Argument of type 'Promise<any[]>' is not assignabl... Remove this comment to see the full error message
         .fetchSession(formPromise, true)
         .then(([session, steps]) => {
+          initState.sessions[formName] = session;
```

Comment:
remove this. should be able to store completed form boolean in the context. rename sessions to cachedSessions